allensdk/test/brain_observatory/behavior/data_objects/running_speed/test_running_speed.py

```diff
@@ -213,19 +257,21 @@ def test_running_speed_from_json(
 )
 def test_running_speed_to_json(
```

Comment:
Why doesn't this one get the pytest skip?

Reply:
It did -- it's above the huge set of test parameters